support/cas-server-support-actions/src/main/java/org/apereo/cas/web/flow/GenerateServiceTicketAction.java

```diff
@@ -75,18 +74,6 @@ protected Event doExecute(final RequestContext context) {
                         + ticketGrantingTicket), ticketGrantingTicket);
             }
 
-            final RegisteredService registeredService = servicesManager.findServiceBy(service);
```

Comment:
I dont think this bit is redundant. If a ticket is about to be generated for a service is no longer authorized to access CAS while an SSO session does exist already, (and the deny was applied to the service config after the fact), are there checks in place to make sure access is denied?

Reply:
Good point, there is a window for the service config to change. The time between getting redirected to the login form and actually submitting the form.

I'll put this back in and add `ServiceSelectionPlan` support to it.

Reply:
Excellent. Thanks!